platform/30-kyverno-setup.sh

```diff
@@ -50,7 +50,7 @@ kubectl create secret generic regcred --type=kubernetes.io/dockerconfigjson --fr
 ### failed calling webhook "mutate.kyverno.svc-fail" error. Once this image
 ### is offically released, this will be removed.
 echo -e "${C_GREEN}Patching Kyverno image until release of v1.6.0...${C_RESET_ALL}"
-kubectl set image -n kyverno deploy/kyverno kyverno=ghcr.io/jimbugwadia/kyverno:v1.6.0-130-g8a0d465d
+kubectl set image -n kyverno deploy/kyverno kyverno=ghcr.io/jimbugwadia/kyverno:v1.6.0-latest
```

Comment:
Would something like the following work to pin the version of this image?
```
kubectl set image -n kyverno deploy/kyverno kyverno=ghcr.io/jimbugwadia/kyverno@sha256:c9f1655e61dfc043e37d63c960b762c118485465b4c3def0dacb2d13cf7e4dca
```

Reply:
Yes we can do that. As Jim does not update his own image regularly maybe its not needed? @mlieberman85 was saying Kyverno full release was coming in January. What does the team think?